# viralnet

[![Gitter](https://badges.gitter.im/viralnet/community.svg)](https://gitter.im/viralnet/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)

### Project data

* **all_repeats.fna.gz**                                                  (sha1sum:17062595b3d81a16e26980847fe6be409a6d8cdb) : All CRISPR repeats by `TAXON_OID___SCAFFOLD_OID`
* **meta_spacers.fna.gz**                                                 (sha1sum:05419668e4a795bafbbdf780dda7462fb6f1879f) : All CRISPR repeats by `TAXON_OID:SCAFFOLD_OID:???:???`
* **spacers_vs_all_viral_sequences_combined_v2_Filtered_1Snp_100AF.lout** (sha1sum:af4263106836c6773ef7f82ace24a3a35a2d67c7) : LAST alignment between all spacers and viral contigs [1-4]
* **Table1_5_combined.txt**                                               (sha1sum:cb2e24ccd384209f8ce4260398ff660b29c22c41) : Metadata on viral contigs and viral OTUs [1-4]
* **viral_spacers.fna**                                                   (sha1sum:fbfe6476ebf2b0339a9993df84a54a81a8299843) : All CRISPR spacers by `TAXON_OID:SCAFFOLD_OID:???:???`
* **predicted_hosts.tsv**                                                 (sha1sum:4918fd93dff34af8423e067c720f029529a4f6d1) : Host predictions for viral UViGs [1-4]

### Project notes

* Because this project makes heavy use of Jupyter notebooks, we recommend using
  [`nbdime`](https://nbdime.readthedocs.io/en/stable/index.html) to make
  interaction with git more convenient.

### References

[1] Paez-Espino, D., Roux, S., Chen, I.M.A., Palaniappan, K., Ratner, A., Chu, K., Huntemann, M., Reddy, T.B.K., Pons, J.C., Llabrés, M. and Eloe-Fadrosh, E.A., 2018. **IMG/VR v. 2.0: an integrated data management and analysis system for cultivated and environmental viral genomes.** *Nucleic acids research*, [DOI : 10.1093/nar/gky1127](https://doi.org/10.1093/nar/gky1127).

[2] Paez-Espino, D., Pavlopoulos, G.A., Ivanova, N.N. and Kyrpides, N.C., 2017. **Nontargeted virus sequence discovery pipeline and virus clustering for metagenomic data.** *Nature Protocols*, [DOI : 10.1038/nprot.2017.063](https://doi.org/10.1038/nprot.2017.063).

[3] Paez-Espino, D., Eloe-Fadrosh, E.A., Pavlopoulos, G.A., Thomas, A.D., Huntemann, M., Mikhailova, N., Rubin, E., Ivanova, N.N. and Kyrpides, N.C., 2016. **Uncovering Earth’s virome.** *Nature*, [DOI : 10.1038/nature19094](https://doi.org/10.1038/nature19094).

[4] Paez-Espino, D., Chen, I.M.A., Palaniappan, K., Ratner, A., Chu, K., Szeto, E., Pillay, M., Huang, J., Markowitz, V.M., Nielsen, T. and Huntemann, M., 2016. **IMG/VR: a database of cultured and uncultured DNA Viruses and retroviruses.** *Nucleic acids research*, [DOI : 10.1093/nar/gkw1030](https://doi.org/10.1093/nar/gkw1030).

[5] Chen, I.M.A., Chu, K., Palaniappan, K., Pillay, M., Ratner, A., Huang, J., Huntemann, M., Varghese, N., White, J.R., Seshadri, R. and Smirnova, T., 2018. **IMG/M v. 5.0: an integrated data management and comparative analysis system for microbial genomes and microbiomes.** *Nucleic acids research*, [DOI : 10.1093/nar/gky901](https://doi.org/10.1093/nar/gky901).
